modules/log/tests/src/Kernel/OrderIntegrationTest.php

```diff
@@ -190,4 +193,25 @@ public function testPlaceValidateFulfillLogs() {
     $this->assertText('The order was fulfilled.');
   }
 
+  /**
+   * Tests that an order assignment log is generated.
+   */
+  public function testOrderAssignedLog() {
+    // Reassignment does not work on users
```

Comment:
The comment is confusing (doesn't work - why?). I suggest "Reassignment is currently only done on user login."